docs/UserGuide.md

```diff
@@ -117,64 +117,151 @@ Examples:
 
 Shows a list of all the students in the student list.
 
-![list students](images/liststu.png)
+![list students](images/userguide-screenshots/liststu.png)
 
 Format: `liststu`
 
-### Editing a person : `edit`
+### Adding attendance : `attendance`
 
-Edits an existing person in the address book.
+Increase or decrease the number of times a student attended tutorials.
```

Comment:
A trivial thing here. Should it be changed to increases or decreases? I saw the others are all this form.